include/picongpu/initialization/ParserGridDistribution.cpp

```diff
@@ -25,8 +25,7 @@
 #include <string>   // std::string
 #include <iterator> // std::distance
 
-#include <boost/regex.hpp>
-#include <boost/lexical_cast.hpp>
+#include <regex>
```

Comment:
bye, byte GCC 4.8 :stars: 
(`INSTALL.md` already lists GCC 4.9+, so that's fine.)